src/modules/new-like.js

```diff
@@ -0,0 +1,30 @@
+async function newLike(id) {
+  const apikey = 'LHYarZybqm9V0G7OV772';
+  const url = `https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/${apikey}/likes/`;
+
+  try {
+    const response = await fetch(url, {
+      method: 'POST',
+      body: JSON.stringify({
+        item_id: id,
+        likes: 1,
+      }),
+      headers: {
+        'Content-type': 'application/json; charset=UTF-8',
+        Accept: 'application/json',
+      },
+    });
+
+    if (!response.ok) {
+      throw new Error('Error adding new like');
+    }
+
+    // Optionally, return the parsed JSON response
+    return response.json();
+  } catch (error) {
+    console.error('Error adding new like:', error);
+    return { error: 'Error adding new like' }; // Return an error object
+  }
+}
+
+export default newLike;
```

Comment:
When clicking on the Like button, it updates the UI but when I refresh the page, it is back to the original number. Have a look at the below recording and try to fix the issue.

https://github.com/Zuwairanajma/Javascript-Capstone_AlbertandJuw/assets/33268927/ed2f776c-bbd4-4621-bce1-534da994645f